app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -196,6 +196,26 @@ protected function _getNormalizeHostTarget() {
     }
 
     /**
+     * Get hosts as regex
+     *
+     * ex: base_url: example.com
+     *     path_regex: (example.com|example.net)
+     *
+     * @return string
+     */
+    public function getAllowedHostsRegex() {
+    	$hosts = array();
+    	foreach( Mage::app()->getStores() as $store ) {
+    		$hosts[] = parse_url( $store->getBaseUrl( Mage_Core_Model_Store::URL_TYPE_WEB , false ), PHP_URL_HOST );
+    	}
+    	 
+    	$hosts = array_values(array_unique( $hosts ));
+    	 
+    	$pattern = '('.implode('|', $hosts).')';
```

Comment:
@felixbuenemann would you mind creating a PR with your change? Have you faced a case where this is not working?